src/hooks/useDailyStatistics/useDailyStatistics.tsx

```diff
@@ -4,10 +4,12 @@ import { CampaignConfigContext } from "../../context/campaignConfig";
 import { getDailyStatistics } from "../../services/statistics";
 import { countTotalTransactionsAndByCategory } from "./utils";
 import { sortTransactionsByOrder } from "../../components/CustomerQuota/utils";
+import { useTranslate } from "../useTranslate/useTranslate";
 
 export type StatisticsHook = {
   totalCount: number | null;
   lastTransactionTime: Date | null;
+
```

Comment:
Nit: please remove this new line as well

Reply:
done